src/web_app/js/call.js

```diff
@@ -419,23 +419,45 @@ Call.prototype.maybeCreatePcClient_ = function() {
   if (this.pcClient_) {
     return;
```

Comment:
Well spotted, I think we do reuse when we rejoin the call but have not confirmed this 100%.